stories/index.js

```diff
@@ -3,8 +3,8 @@ import { storiesOf, action, linkTo } from '@kadira/storybook';
 import { host } from 'storybook-host';
 
 import SnoozePopup from '../src/lib/components/SnoozePopup';
-import MainPanel from '../src/lib/components/MainPanel';
-import ManagePanel from '../src/lib/components/ManagePanel';
+// import MainPanel from '../src/lib/components/MainPanel';
+// import ManagePanel from '../src/lib/components/ManagePanel';
```

Comment:
I'll merge whatever you have up at the end of this meeting.  Either would be fine.  🙂